### PR TITLE
Apply changes (including local highscore) to CoronaJump

### PR DESCRIPTION
**Pull request by tguelenman**
as stated in https://github.com/abegehr/monsterjump/pull/54#issuecomment-611491327

### CODE REVIEW

**Comment by tguelenman** [comment]
@abegehr how to proceed here?

File: lib/components/player.dart [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import 'dart:async';
+//import 'dart:html'; // TODO move to web only.
```
Comment (abegehr) [inline lib/components/player.dart:2]: web stuff should not be commented out

Comment (tguelenman) [reply lib/components/player.dart:2]: @abegehr this is not used. what exactly should be done here? delete the line completely?

Comment (abegehr) [reply lib/components/player.dart:2]: this is used on web, but causes crashes on native. See #67

**Comment by abegehr** [comment]
@tguelenman this is still open.

**Comment by tguelenman** [comment]
> @tguelenman this is still open.

@abegehr you did not answer my question:

> what exactly should be done here?

**Comment by abegehr** [comment]
> > @tguelenman this is still open.
> 
> 
> 
> @abegehr you did not answer my question:
> 
> 
> 
> > what exactly should be done here?

The goal is to make local Highscore work on CoronaJump.

**Comment by tguelenman** [comment]
@abegehr as far as I'm concerned you still did not answer my question

> what exactly should be done here?

If you don't know what should be done to reach the goal

> to make local Highscore work on CoronaJump

then please say so, so that I'm not waiting on an answer

**Comment by abegehr** [comment]
done in #72

**Pull request closed without merging**